src/sql/workbench/services/notebook/browser/sql/sqlSessionManager.ts

```diff
@@ -382,6 +382,10 @@ class SqlKernel extends Disposable implements nb.IKernel {
 		});
 	}
 
+	restart(): Thenable<void> {
+		return Promise.reject(localize('SqlKernelRestartNotSupported', 'SQL kernel restart not supported'));
```

Comment:
Are you planning on adding this in later? Could just be disconnecting/reconnecting the current connection

Reply:
In the doc I wrote, I put this as a no-op, so wasn't planning on adding this now. I can look at whether disconnect/connect is as simple as two method calls, but if it becomes more complicated than that, I'd prefer not to tackle this immediately (since users can easily use the attach to to change connection).

Reply:
Yeah, definitely worth doing in a separate PR. Just wanted to throw it out there while I was thinking about it